src/pages/johyeoungjin/Login/JoLogin.js

```diff
@@ -1,7 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import './JoLogin.scss';
 
 function JoLogin() {
-  return <div>조형진 로그인페이지</div>;
+  const [idValue, setIdValue] = useState('');
+  const [pwValue, setPwValue] = useState('');
+  const navigate = useNavigate();
+
+  const saveUserId = event => {
+    console.log(idValue);
+    setIdValue(event.target.value);
+  };
+  const saveUserPw = event => {
+    console.log(pwValue);
+    setPwValue(event.target.value);
+  };
```

Comment:
```suggestion
  const saveUserId = event => {
    const { name, value } = event.target;
    setUserInput({ ...userInput, [name] : value });
  };
```
- 구조 분해 할당과 계산된 속성명을 활용한다면 하나의 함수로도 관리가 가능합니다.